ci-operator/step-registry/ipi/conf/vsphere/permission/ipi-conf-vsphere-permission-commands.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+declare vcenter_username_permission
+declare vcenter_password_permission
+
+VCENTER_AUTH_PATH=/var/run/vault/devqe-secrets/secrets.sh
+# shellcheck source=/dev/null
+source "${VCENTER_AUTH_PATH}"
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/permission-user.yaml.patch"
+
+cat > "${PATCH}" << EOF
+platform:
+  vsphere:
```

Comment:
@jinyunma do you still remember this is the credential where installer is used to create resources on vsphere platform, does not need any separate local client credential file?

Reply:
Do you mean client certificates files? It is independent with user and fixed per vsphere instance.

Reply:
e.g: in azure, we need to create a dedicated sp file, and export it via some ENV var, so that installer can use it to create resources on azure platform, so I wonder vpshere is not that case, just directly use the user/password in install-config.yaml to create those resources.

Reply:
Get your point. Yes, credentials are directly set in install-config for vsphere, don't require separate file for this.

Reply:
Got it, thanks for confirming.